pkg/server/node.go

```diff
@@ -245,6 +247,12 @@ var (
 		`interval at which the liveness range is compacted. A value of 0 disables the periodic compaction`,
 		0,
 	)
+	perConsumerCatchupLimit = settings.RegisterBoolSetting(
+		settings.SystemOnly,
+		"kv.rangefeed.per_consumer_catchup_scan_limit.enabled",
+		"if enabled, rangefeed catchup scans are limited on a per consumer basis",
+		true,
```

Comment:
In DR weekly with @dt, we are of the opinion that this should merge with default = false, and keep this in our back pockets in case we need it to bail us out of a production situation.  @stevendanna does that make sense to you?

Reply:
> I've grown very skeptical of this change over the last few days of investigations.

Could you elaborate?

Reply:
> Could you elaborate?

Overall, this change doesn't feel very risky to me and may be a useful mitigation for some users; but, it comes at the cost of adding another layer of confusion.  There are currently 2 rate limiters and CPU pacing to consider when thinking about a backlog of catchup scans.  Recently, it took multiple CRDB developers a non-negligible amount of time to fully understand that the joint behaviour of the _existing_ rate limiters.  

Further, we found that since the introduction of these rate limiters, the impact of catch-up-scans on foreground workloads has improved dramatically.  We recently ran an experiment where we set both limits to 35k and saw only minor impacts on the foreground workload.  This isn't an argument for removing the limits immediately (since there are other problems with having up to 35k iterators open), but it is perhaps just more motivation to re-visit the overall structure of the catch-up scan process and the limits around it.